pkg/api/server/server.go

```diff
@@ -14,8 +14,25 @@ import (
 	"github.com/obot-platform/obot/pkg/proxy"
 	"github.com/obot-platform/obot/pkg/storage"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apiserver/pkg/authentication/user"
 )
 
+// authDisabledPaths is a list of paths that do not require authentication.
+var (
```

Comment:
We will have to disable authenticate for webhook, as sendgrid doesn't have a way to configure the webhook secrets.

For other providers it might support it, but I think it will be a total different workflow to configure token in obot as supposed to having token dedicated for webhook usage. So, no auth for now.